src/cpp/README.md

```diff
@@ -7,3 +7,5 @@ This directory contains source code for C++ implementation of gRPC.
 
 Alpha : Ready for early adopters
 
+#Reference
+[gRPC Documentation](https://github.com/grpc/grpc-common)
```

Comment:
We're actually moving most of the user docs (other than installation instructions/release notes that live in the individual language repos) into our public site at grpc.io, can you change the link to http://www.grpc.io/docs/? Thanks!